addons/stock_account/__init__.py

```diff
@@ -39,3 +39,4 @@ def _configure_journals(env):
             data['account.journal'] = ChartTemplate._get_stock_account_journal(template_code)
```

Comment:
I would suggest you edit the `data` above by doing something like
```py
        data = {
            'template_data': {
                fname: value
                for fname, value in full_data['template_data'].items()
                if fname in [
                    'property_stock_journal',
                    'property_stock_account_input_categ_id',
                    'property_stock_account_output_categ_id',
                    'property_stock_valuation_account_id',
                ]
            },
            'res.company': {
                company.id: {
                    fname: value
                    for fname, value in full_data['res.company'][company.id]
                    if fname in ('account_production_wip_account_id', 'account_production_wip_overhead_account_id')
                }
            }
        }
```
so the fields on `res.company` get written during the call to `_load_data`